.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
Is this repo wide?

Reply:
Yes, thought there should be one, but I can remove if you don't want it there